terraform/templates/validator.json

```diff
@@ -43,6 +43,9 @@
             {"name": "CFG_NUM_VALIDATORS_IN_GENESIS", "value": "${cfg_num_validators_in_genesis}"},
             {"name": "CFG_SEED", "value": "${cfg_seed}"},
             {"name": "CFG_SEED_PEER_IP", "value": "${cfg_seed_peer_ip}"},
+        %{ if slog_path != "" }
```

Comment:
@sherry-x  is this a typo for `structlog_path`? Running `terraform plan --var-file=facebook/dev.tfvars` fails with:

```
Error: failed to render : <template_file>:46,15-24: Unknown variable; There is no variable named "slog_path".

  on validators.tf line 192, in data "template_file" "ecs_task_definition":
 192: data "template_file" "ecs_task_definition" {
...
```